ui/app/pages/home/home.container.js

```diff
@@ -3,7 +3,7 @@ import { compose } from 'recompose'
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import { unconfirmedTransactionsCountSelector } from '../../selectors/confirm-transaction'
-``
+
```

Comment:
Remove these changes

Reply:
This change is to remove as dangling template string out of no where. Why do we not want to remove them?

Reply:
I've moved this to a separate PR so that we can also add an eslint rule to prevent this from happening again: https://github.com/MetaMask/metamask-extension/pull/6839/files